api/HTMLMediaElement.json

```diff
@@ -357,7 +357,7 @@
               "version_added": "4.0"
             },
             "webview_android": {
-              "version_added": "43"
+              "version_added": true
```

Comment:
Why this change? A lot of the "43" in this file are wrong and would be fixed by testing the instance, maybe fix all of those first to ensure that no known-wrong data is being mirrored or overwritten in this PR? We should be able to fill in a real value here after the 43s and true values elsewhere are fixed.

Reply:
I actually don't know why I did this.  I think I might've been thinking "I'll set it to `true` and fix these later", but we've got an issue open to track these down.  Reverting these changes so that this PR is only increasing the amount of real values, not decreasing.

Reply:
Yep, let's get back to these in https://github.com/mdn/browser-compat-data/issues/7843.